irispy/iris_tools.py

```diff
@@ -19,13 +19,12 @@
 
 # Define some properties of IRIS detectors.  Source: IRIS instrument
 # paper.
-DETECTOR_GAIN = {"NUV": 18., "FUV": 6., "SJI": 18.}
-DETECTOR_YIELD = {"NUV": 1., "FUV": 1.5, "SJI": 1.}
+DETECTOR_GAIN = {"NUV": 18., "FUV1": 6., "FUV2": 6., "SJI": 18.}
+DETECTOR_YIELD = {"NUV": 1., "FUV1": 1.5, "FUV2": 1.5, "SJI": 1.}
```

Comment:
The reason there is only one an FUV key in this dictionary, rather than an FUV1 and FUV2 is that it reflects the metadata supplied by the IRIS instrument team and other aspects of the metadata may only be labelled FUV.  Not everything is labelled FUV 1 or FUV2.  So I think this should stay FUV for the time being.  When determining what detector type to use, when can do something like ```if "FUV" in detector_type```.  This will give us the right mapping whether detector_type is "FUV", "FUV1" or "FUV2".